flutter_local_notifications/lib/src/platform_specifics/android/icon.dart

```diff
@@ -1,12 +1,20 @@
+import 'dart:convert';
+import 'dart:typed_data';
+
+import 'enums.dart';
+
 /// Represents an icon on Android.
-abstract class AndroidIcon {
+abstract class AndroidIcon<T> {
```

Comment:
Thanks, these changes make sense and makes it consistent with how bitmaps are handled. They are however, breaking changes and with Android 12 release is around the corner. Do you think you could try to make this PR target the 9.0.0 branch instead so I can try to bundle it with the 9.0.0 release?

Reply:
rebased and changed target

Reply:
Is this what you wanted?

Reply:
Yep

Reply:
Thanks :)